test/data/fle2-find-range-explicit/double-precision/value-to-encrypt.json

```diff
@@ -0,0 +1,20 @@
+{
+    "v": {
+        "$and": [
+            {
+                "age": {
+                    "$gte": {
+                        "$numberDouble": "123.456"
+                    }
+                }
+            },
+            {
+                "age": {
```

Comment:
This might be related more to a testing ticket, and not this ticket, but it could be helpful to create a json test that has an open range (like just a `$gte` and not both `$lte` and `$gte`)?